### PR TITLE
Fix survey invites in IE

### PR DESCRIPTION
**Pull request by lucid-0**

### CODE REVIEW

**Comment by zombified** [comment]
@lucid-0 please recompile the conflicting files

**Comment by vangheem** [comment]
Sorry, needs a rebase.

File: castle/cms/static/patterns/survey.js [hunk 1]
```diff
@@ -113,7 +113,8 @@ define([
       try {
         xhr.send(data);
         xhr.onload = function() {
-          self.renderInvite(self.survey_tile, self.survey_data, xhr.response);
+          self.invite_data = JSON.parse(xhr.response);
```
Comment (vangheem) [inline castle/cms/static/patterns/survey.js]: xhr.response is only not an object for IE? Can we be sure this doesn't break non-IE?

Maybe even wrap in try/catch?

**Comment by vangheem** [comment]
@lucid-0 sorry to bug you about this again. Can you update, build resources and add a changelog entry?

**Pull request merged**